common/utils/storage_backends.py

```
@@ -0,0 +1,9 @@
+from storages.backends.s3boto3 import S3Boto3Storage
+
+class MediaStorage(S3Boto3Storage):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)